### ISSUE TITLE
Don't unblock inflight requests when the last intecept is removed

### ISSUE THREAD

**Issue by Lightning00Blade** [GoogleChromeLabs/chromium-bidi#1890]
Consider the following scenario:

1. Enable interception for URL A
2. Fetch for URL A, it gets Blocked
3. Remove interception
4. Expectation:

    -  The request should continue to be blocked
5. Reality:

    -  The request is continued

**Comment by Lightning00Blade** [comment]
Closing in favor of https://github.com/GoogleChromeLabs/chromium-bidi/issues/2147
